### PR TITLE
Adds phpcs npm command

### PR DESCRIPTION
**Pull request by belcherj**
Adds npm command to run phpcs: `npm run phplint`

### Detailed test instructions:

 - Run `npm run phplint` at the command line

### CODE REVIEW

**Review by joshuatf — COMMENTED** [review]
Sweet-nice addition!

I am curious why running this shows a bunch of warnings that our pre-commit hook running phpcs doesn't show 🤔 Do you see the same?

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "i18n:pot": "grunt makepot",
     "i18n:check": "grunt checktextdomain",
     "i18n": "npm run -s i18n:js && npm run -s i18n:php && npm run -s i18n:check && npm run -s i18n:pot",
+    "phplint": "./vendor/bin/phpcs --standard=phpcs.xml.dist $(git ls-files | grep .php$)",
```
Comment (ryelle) [inline package.json]: Can we call this `lint:php` to follow the other commands (like `i18n`)? Maybe rename `lint` to `lint:js`, and then have `lint` run both commands?

**Review by ryelle — COMMENTED** [review]
Left a comment about naming, otherwise this looks good 👍

**Comment by ryelle** [comment]
> I am curious why running this shows a bunch of warnings that our pre-commit hook running phpcs doesn't show 🤔 Do you see the same?

Looks like we run it with `-n` to hide warnings in the pre-commit hook, so only errors would prevent commit.

**Pull request merged**